### PR TITLE
Move Gossip values added for wen_retart into restart_crds_values.

### PR DESCRIPTION
**Pull request by wen-coding**
#### Problem
Group Gossip values added for wen_restart into one single file.

#### Summary of Changes
Move RestartLastVotedForkSlots into wen_restart_values

### CODE REVIEW

File: gossip/src/wen_restart_values.rs
Comment (behzadnouri) [inline gossip/src/wen_restart_values.rs:1]: Can we name this file `restart_crds_values.rs`?

File: gossip/src/wen_restart_values.rs [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    }
+}
+
+#[derive(Debug, Error)]
+pub enum RestartLastVotedForkSlotsError {
+    #[error("Last voted fork cannot be empty")]
+    LastVotedForkEmpty,
+}
+
+impl RestartLastVotedForkSlots {
```
Comment (behzadnouri) [inline gossip/src/wen_restart_values.rs:115]: Can you please move all `enum/struct` type definitions first in the file, and then followed by `impl`?

Also please put more prominent types first, then the inner/private types; e.g. `RestartLastVotedForkSlots` should go first.

File: gossip/src/wen_restart_values.rs
Comment (wen-coding) [reply gossip/src/wen_restart_values.rs:1]: Done.

File: gossip/src/wen_restart_values.rs (hunk 1 above)
Comment (wen-coding) [reply gossip/src/wen_restart_values.rs]: Done.

**Review by behzadnouri — DISMISSED** [review]

**Review by behzadnouri — APPROVED** [review]

**Review by behzadnouri — APPROVED** [review]

**Pull request merged**